sdk/identity/azure-identity/src/test/java/LiveManagedIdentityTests.java

```diff
@@ -69,8 +69,11 @@ public void testManagedIdentityAksDeployment() {
         //Setup Env
         Configuration configuration = Configuration.getGlobalConfiguration().clone();
 
+
+
+
         String spClientId = configuration.get("IDENTITY_CLIENT_ID");
-        String secret = configuration.get("IDENTITY_CLIENT_SECRET");
+        String oidc = configuration.get("ARM_OIDC_TOKEN");
```

Comment:
If you want to use this token at the running of the tests you will also need to set it as an env variable in the tests.yml file.